Validators.js

```diff
@@ -112,4 +112,39 @@ function validateUserDataValue(stringValue, stringName, functionName) {
     return validateParseInt(stringValue, stringName, functionName);
 }
 
-export { validateUserDataValue as UserDataValue, validateString as String, validateParseInt as ParseInt, validateValidUserData as ValidUserData, validateUserDataType as UserDataType };
+/**
+ * Validate event module parameters for undefined or wrong type.
+ * It will log a message if any issue found and return false.
+ * If the parameters are valid, it will return true.
+ * 
+ * @param {String} functionName : name of function that called this method.
+ * @param {string} eventName - Name of the event.
+ * @param {Segmentation} segments - segementation data for the event.
+ * @param {number} eventCount - event count.
+ * @param {number} eventSum - event sum.
+ * @returns
+ */
+function areEventParametersValid(functionName, eventName, segments, eventCount, eventSum) {
+    if (!eventName || typeof eventName !== 'string') {
+        L.d(`${functionName}, eventName: [${eventName}] must be a string`);
+        return false;
+    }
+    if (segments && typeof segments !== 'object') {
+        L.d(`${functionName}, segments: [${segments}] must be an object`);
+        return false;
+    }
+
+    if (eventCount && typeof eventCount !== 'number') {
```

Comment:
maybe you can validate that all number fields should be positive such as:
- count is ge 1 
- sum is ge 0.0